src/NovelRT/Windowing/WindowingService.cpp

```diff
@@ -135,9 +135,9 @@ namespace NovelRT::Windowing {
   void WindowingService::checkForOptimus(const char* library) {
     _optimus = LoadLibrary(reinterpret_cast<LPCSTR>(library));
     if (_optimus != nullptr) {
-      _logger.logInfoLine("NVIDIA GPU detected. Enabling...");
+      _logger.logInfo("NVIDIA GPU detected. Enabling...");
     } else {
-      _logger.logInfoLine("NVIDIA GPU not detected. Continuing w/o Optimus support.");
+      _logger.logInfo("NVIDIA GPU not detected. Continuing w/o Optimus support.");
```

Comment:
Since were here, you could change that `w/o` to `without`?

Reply:
Done!